wpilib/wpilib/digitalsource.py

```diff
@@ -16,7 +16,7 @@
 __all__ = ["DigitalSource"]
 
 
-def _freeDigitalSource(handle):
+def _freeDigitalSource(handle: hal.DigitalHandle) -> None:
```

Comment:
Given that this is an internal function, I'm not too sure this was necessary.

Reply:
Makes mypy a little happier. And also makes reading the code easier